Initial checkin for Azure Managed Apps sample (#1)
Azure-Samples/communication-services-azure-managed-apps: README.md

# Azure Managed Application samples

## Abstract

> This repository contains Azure Managed Applications samples for Azure Communication Services, that can be used as reference for creating and consuming managed applications for Azure Communication services.

Reviewer: ```suggestion
This repository contains Azure Managed Applications samples for Azure Communication Services. Use these samples as a reference for creating and consuming managed applications for Azure Communication Services.
```

## Documentation

[Azure Communication Services overview](https://learn.microsoft.com/en-us/azure/communication-services/overview)

[Azure Managed Application overview](https://docs.microsoft.com/en-us/azure/azure-resource-manager/managed-application-overview)

[Create and publish Azure Managed Applications to Service Catalog](https://docs.microsoft.com/en-us/azure/azure-resource-manager/managed-application-publishing)

[Create and publish Azure Managed Applications to Azure Marketplace](https://docs.microsoft.com/en-us/azure/azure-resource-manager/managed-application-author-marketplace)

[Consume an Azure Managed Application](https://docs.microsoft.com/en-us/azure/azure-resource-manager/managed-application-consumption)

[Create UI definitions](https://docs.microsoft.com/en-us/azure/azure-resource-manager/managed-application-createuidefinition-overview)

[Azure Managed Application samples](https://github.com/Azure/azure-managedapp-samples/)

## Contributing

> To contribute and get started, please visit our [**contribution guide**](./1-contribution-guide/README.md#contribution-guide).

Reviewer: ```suggestion
To contribute and get started, see the [**Contribution Guide**](./1-contribution-guide/README.md#contribution-guide).
```
